docs/_config.yml

```diff
@@ -11,7 +11,11 @@ github_username: auth0
 github_repo: auth0-cli
 
 # Build settings
-theme: minima
+remote_theme: just-the-docs/just-the-docs@v0.4.0.rc3
```

Comment:
**Request:** Looks like `v0.4.0.rc4` has since come out. I'd considering upgrading.

**Question:** Was using a remote theme over a regular gem a conscious decision?

Reply:
We reverted back to rc3 in 3511d081376b1d1d3cae7f2e4e75a2a31578f8d0 as we can't use rc4 due to an issue with the spacing on the search contents, that's [fixed](https://github.com/just-the-docs/just-the-docs/pull/1123) but yet to be released.

It has to use a `remote_theme` due to building on GitHub pages which [supports limited themes](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll#:~:text=To%20use%20any%20other%20Jekyll%20theme%20hosted%20on%20GitHub%2C%20type%20remote_theme%3A%20THEME%2DNAME%2C%20replacing%20THEME%2DNAME%20with%20the%20name%20of%20the%20theme%20as%20shown%20in%20the%20README%20of%20the%20theme%27s%20repository.) and doesn't run a `bundle install` so the `jekyll-remote-theme` plugin downloads and configures the theme during the jekyll build process instead

Reply:
Ok cool. Just need to remind ourselves to update to the stable version when appropriate.